pkg/clients/ovirt/impl.go

```diff
@@ -25,67 +23,17 @@ import (
 	ovirtconfigv1 "github.com/openshift/cluster-api-provider-ovirt/pkg/apis/ovirtprovider/v1beta1"
 )
 
-type InstanceService struct {
-	Connection   *ovirtsdk.Connection
-	ClusterId    string
-	TemplateName string
-	MachineName  string
-}
-
-type Instance struct {
-	*ovirtsdk.Vm
-}
-
-type SshKeyPair struct {
-	Name string `json:"name"`
-
-	// PublicKey is the public key from this pair, in OpenSSH format.
-	// "ssh-rsa AAAAB3Nz..."
-	PublicKey string `json:"public_key"`
-
-	// PrivateKey is the private key from this pair, in PEM format.
-	// "-----BEGIN RSA PRIVATE KEY-----\nMIICXA..."
-	// It is only present if this KeyPair was just returned from a Create call.
-	PrivateKey string `json:"private_key"`
-}
-
-type InstanceListOpts struct {
-	Name string `json:"name"`
-}
-
-func NewInstanceServiceFromMachine(machine *machinev1.Machine, connection *ovirtsdk.Connection) (*InstanceService, error) {
-	machineSpec, err := ovirtconfigv1.MachineSpecFromProviderSpec(machine.Spec.ProviderSpec)
-	if err != nil {
-		return nil, err
-	}
-
-	service := &InstanceService{Connection: connection}
-	service.ClusterId = machineSpec.ClusterId
-	service.TemplateName = machineSpec.TemplateName
-	service.MachineName = machine.Name
-	return service, err
-}
+var _ OvirtClient = (*ovirtClient)(nil)
```

Comment:
What purpose does this serve? Please document, as it seems to be a hack!

Reply:
This is an interface check https://golang.org/doc/effective_go#blank_implements.
I don't know if this is a convention or just because I'm used to see it, but it's not a must